src/allIdentical.js

```diff
@@ -0,0 +1,32 @@
+import { uniqWith, identical, pipe } from 'ramda';
+import curry1 from 'ramda/src/internal/_curry1';
+
+import lengthLte from './lengthLte';
+
+/**
+ * Returns true if all items in the list are equivalent using `R.identical` for equality comparisons.
+ *
+ * @func allIdentical
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.11.0|v2.11.0}
+ * @category List
+ * @sig [a] -> Boolean
+ * @param {Array} list The list of values
+ * @return {boolean}
+ * @see {@link https://ramdajs.com/docs/#identical|identical}
+ * @example
+ *
+ * RA.allIdentical([ 1, 2, 3, 4 ]); //=> false
+ * RA.allIdentical([ 1, 1, 1, 1 ]); //=> true
+ * RA.allIdentical([]); //=> true
+ * RA.allIdentical([ {}, {} ]); //=> false
+ *
+ */
+const allIdentical = curry1(
```

Comment:
Maybe we can make a more optimized version using `R.reduce` and `R.reduced` ?

On the other hand, the implementation is less beautiful.

Reply:
We could as  a performance optimization after it is in master.